core/src/main/scala/akka/persistence/cassandra/ConfigSessionProvider.scala

```diff
@@ -186,27 +168,31 @@ class ConfigSessionProvider(system: ActorSystem, config: Config) extends Session
    *
    * @param clusterId the configured `cluster-id` to lookup
    */
-  def lookupContactPoints(clusterId: String)(implicit ec: ExecutionContext): Future[immutable.Seq[InetSocketAddress]] = {
+  def lookupContactPoints(
+      clusterId: String
```

Comment:
hm

Reply:
Not a fan? It broke the line due to length, i think just moving the second parameter list to the second line is how i'd have done it manually